src/document/Document.js

```diff
@@ -675,10 +675,12 @@ define(function (require, exports, module) {
     
     /**
      * Updates the language to match the current mapping given by LanguageManager
+     * 
+     * @param {!string} rawText  Text content of the file.
      */
-    Document.prototype._updateLanguage = function () {
+    Document.prototype._updateLanguage = function (rawText) {
```

Comment:
... and yes rawText can be null, so `_updateLanguage()` can be called without `rawText`